scripts/release/download-experimental-build.js

```diff
@@ -11,18 +11,13 @@ const {
 
 const checkEnvironmentVariables = require('./shared-commands/check-environment-variables');
 const downloadBuildArtifacts = require('./shared-commands/download-build-artifacts');
-const getLatestMasterBuildNumber = require('./shared-commands/get-latest-master-build-number');
 const parseParams = require('./shared-commands/parse-params');
 const printSummary = require('./download-experimental-build-commands/print-summary');
 
 const run = async () => {
   try {
     addDefaultParamValue('-r', '--releaseChannel', 'experimental');
-    addDefaultParamValue(
-      null,
-      '--build',
-      await getLatestMasterBuildNumber(true)
-    );
+    addDefaultParamValue(null, '--commit', 'master');
```

Comment:
This is a simpler solution but it won't work for the Firefox builds because they need to be reproducible (and `scripts/release/download-experimental-build.js --commit=master` isn't a valid way to repro a specific build). I'll propose a change with a follow up PR though.

Reply:
Hm do we not include the `build-info.json` with CI builds anymore? I was hoping we could read from it instead but I don't see it in the `build2` artifacts.

Reply:
Ooh now I see this:
https://github.com/facebook/react/blob/0e526bcec2a2940eb2cea6204efab76a2e4d7c0d/scripts/release/publish-commands/print-follow-up-instructions.js#L39-L43

But is there a reason why the new build script doesn't/can't output it? We're in a weird space now where our scripts both validate and require that `build-info.json` is in the package `files` array _and_ we no longer publish it 😁

Reply:
Some follow up: #20723

Reply:
> But is there a reason why the new build script doesn't/can't output it?

Figured it was worth seeing if it's actually necessary anymore. Might be nice to stop requiring every package to publish that file to npm, if we can.

I believe the last remaining reason we need it is because the `prepare-release-from-npm` needs it. But maybe we can instead pull the build artifacts from CI, like we do with releases.

Reply:
I guess, but at the same time I don't see much of a downside of publishing it. It adds a little complexity in one place (but we already have it in place and well "tested") in exchange for avoiding it in another (having to pull a duplicate artifact from CI somehow– which seems difficult to do if we don't have build-info)